gradle.properties

```diff
@@ -109,7 +109,7 @@ jetbrains.compose.compiler.version=1.5.14.1
 #   (artifactRedirection.version.androidx.compose covers androidx.compose.*:*)
 artifactRedirection.version.androidx.compose=1.9.0-rc01
 artifactRedirection.version.androidx.compose.material.material-navigation=1.9.0-rc01
-artifactRedirection.version.androidx.compose.material3=1.4.0-alpha18
+artifactRedirection.version.androidx.compose.material3=1.4.0-beta01
```

Comment:
You said that the navigation suite wasn't promoted to beta. This means, before merging this PR, we have to decouple it. I can do that